accepted/2024/NuGet-Audit-for-PackageDownload.md

```diff
@@ -0,0 +1,106 @@
+# ***Enhancing NuGet Audit for Secure and Selective `<PackageDownload>` Vulnerability Reporting***
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author Name [Nigusu](https://github.com/Nigusu-Allehu)
+- [GitHub Issue](https://github.com/NuGet/Home/issues/13658)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+In our effort to strengthen the security of NuGet, we would like to expand the support of NuGet Audit to `<PackageDownload>` packages. This will be done by introducing vulnerability errors for packages downloaded using `PackageDownload`. However, the .Net sdk also makes use of `PackageDownload` to implicitly download packages. And we would like to make sure these packages are not also receiving these errors, in order to prevent customers being confused. This will be done by introducing a new attribute to `PackageDownload` that makes it clear to NuGet Audit it should not write an error for these packages.
```

Comment:
Please put at new line at the end of each sentence, so it's one sentence per line.  This makes it much easier to comment on specific things